x/auth/ante/ante_test.go

```diff
@@ -652,7 +652,7 @@ func TestAnteHandlerFees(t *testing.T) {
 
 // Test logic around memo gas consumption.
 func TestAnteHandlerMemoGas(t *testing.T) {
-	t.Skip("skipping test as not relevant to Heimdall")
+	t.Skip("skipping test as not relevant to Heimdall (no memo)")
```

Comment:
I understand the need at present to do this, however the SDK is meant to be general purpose. If you need functionality, you use/configure it. The concern is that skipping these tests could leave to an oversight down the road if you do decide to mainline your version of the sdk.

Reply:
Gotcha, thanks, I unskipped and edited the tests accordingly. Will push soon.